zh-cn: revise the translation of other embedding technologies

hanyujie2002 opened the pull request:
I am somewhat concerned about copyright

https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L85-L88

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->

<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->
Relates to #18313

hanyujie2002 committed:
zh-cn: revise translation of other embedding technologies

Signed-off-by: hanyujie2002 <yujiehan2002@outlook.com>

hanyujie2002 committed:
Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

hanyujie2002 committed:
fix

Signed-off-by: hanyujie2002 <yujiehan2002@outlook.com>

yin1999 commented:
> I am somewhat concerned about copyright
> 
> https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L85-L88

我想引用[我们在署名和版权许可](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Attrib_copyright_license#%E9%93%BE%E6%8E%A5%E5%88%B0_mdn_web_%E6%96%87%E6%A1%A3%E4%B8%AD%E7%9A%84%E6%96%87%E7%AB%A0)中提到的一句话来回答你：

> 超文本链接不仅是 web 的本质，它既是将你的用户指向宝贵资源的一种方式，也是对我们社区所做工作的信任。

对于视频网站也是类似，不论是 B 站，还是 Youtube 之类的视频网站，都允许使用 HTML 框架（frame），将视频嵌入到其他的网站中。这是一种对资源的引用——开放 Web 的本质。

hanyujie2002 commented:
> > I am somewhat concerned about copyright
> > https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L85-L88
> 
> 我想引用[我们在署名和版权许可](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Attrib_copyright_license#%E9%93%BE%E6%8E%A5%E5%88%B0_mdn_web_%E6%96%87%E6%A1%A3%E4%B8%AD%E7%9A%84%E6%96%87%E7%AB%A0)中提到的一句话来回答你：
> 
> > 超文本链接不仅是 web 的本质，它既是将你的用户指向宝贵资源的一种方式，也是对我们社区所做工作的信任。
> 
> 对于视频网站也是类似，不论是 B 站，还是 Youtube 之类的视频网站，都允许使用 HTML 框架（frame），将视频嵌入到其他的网站中。这是一种对资源的引用——开放 Web 的本质。

https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L264-L266

I was concerned because there is copyright information in the same article, so referring a copyrighted video might seem ironic.

hanyujie2002 committed:
change bilibili link to respect the copyright

Signed-off-by: hanyujie2002 <yujiehan2002@outlook.com>

hanyujie2002 commented:
I change the video to av1, which doesn't exist on Bilibili now, to respect the copyright.

yin1999 commented:
> > > I am somewhat concerned about copyright
> > > https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L85-L88
> > 
> > 
> > 我想引用[我们在署名和版权许可](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Attrib_copyright_license#%E9%93%BE%E6%8E%A5%E5%88%B0_mdn_web_%E6%96%87%E6%A1%A3%E4%B8%AD%E7%9A%84%E6%96%87%E7%AB%A0)中提到的一句话来回答你：
> > > 超文本链接不仅是 web 的本质，它既是将你的用户指向宝贵资源的一种方式，也是对我们社区所做工作的信任。
> > 
> > 
> > 对于视频网站也是类似，不论是 B 站，还是 Youtube 之类的视频网站，都允许使用 HTML 框架（frame），将视频嵌入到其他的网站中。这是一种对资源的引用——开放 Web 的本质。
> 
> https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L264-L266
> 
> I was concerned because there is copyright information in the same article, so referring a copyrighted video might seem ironic.

One thing to reminder, "use resource" means we have copied the content to MDN, not just using frame to refer the video (and this is allowed by Bilibili:

![image](https://github.com/mdn/translated-content/assets/15844309/5111475f-2618-41f6-80b9-b4a4f4175551)

hanyujie2002 commented:
> > > > I am somewhat concerned about copyright
> > > > https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L85-L88
> > > 
> > > 
> > > 我想引用[我们在署名和版权许可](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Attrib_copyright_license#%E9%93%BE%E6%8E%A5%E5%88%B0_mdn_web_%E6%96%87%E6%A1%A3%E4%B8%AD%E7%9A%84%E6%96%87%E7%AB%A0)中提到的一句话来回答你：
> > > > 超文本链接不仅是 web 的本质，它既是将你的用户指向宝贵资源的一种方式，也是对我们社区所做工作的信任。
> > > 
> > > 
> > > 对于视频网站也是类似，不论是 B 站，还是 Youtube 之类的视频网站，都允许使用 HTML 框架（frame），将视频嵌入到其他的网站中。这是一种对资源的引用——开放 Web 的本质。
> > 
> > 
> > https://github.com/mdn/translated-content/blob/4f51f754df7707fd7e8e10b91de842cda0b50dd1/files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md?plain=1#L264-L266
> > 
> > I was concerned because there is copyright information in the same article, so referring a copyrighted video might seem ironic.
> 
> One thing to reminder, "use resource" means we have copied the content to MDN, not just using frame to refer the video (and this is allowed by Bilibili:
> 
> ![image](https://private-user-images.githubusercontent.com/15844309/323150789-5111475f-2618-41f6-80b9-b4a4f4175551.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzNDU3NjgsIm5iZiI6MTcxMzM0NTQ2OCwicGF0aCI6Ii8xNTg0NDMwOS8zMjMxNTA3ODktNTExMTQ3NWYtMjYxOC00MWY2LTgwYjktYjRhNGY0MTc1NTUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE3VDA5MTc0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4MmM3MDhhYmE0NjZlNmNhNDY0MDU5MDRjZTdlZDY1NDgzMDE4MjAyM2ViNjEyYTc4MTE4NzBmN2RkNWExNWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9T0sgZWGimucG2Ystt5ij3g4O-wH8B_S9EVqVzUsJx0)

I see, thank you

hanyujie2002 commented:
It appears that video uploaders retain the majority of rights to their uploaded videos while some rights are granted to the Bilibili platform. This situation raises significant legal concerns, and it is crucial for us to exercise utmost caution. Consequently, I have modified the video URL so that only a video with nonsensical content will be displayed within the iframe.

yin1999 committed:
Update index.md

yin1999 commented:
> It appears that video uploaders retain the majority of rights to their uploaded videos while some rights are granted to the Bilibili platform. This situation raises significant legal concerns, and it is crucial for us to exercise utmost caution. Consequently, I have modified the video URL so that only a video with nonsensical content will be displayed within the iframe.

I still don't understand your reason for doing this. I've explained above that no other right (copyright) is required to do this, so is there any benefit to be gained by replacing it with an invalid video embed?

resources:

- https://m.thepaper.cn/baijiahao_12539877

hanyujie2002 commented:
I apologize if there is any confusion. My concern is that 小可儿, the author of "念诗之王/改革春风吹满地", has not granted us permission yet, even though Bilibili has.

yin1999 commented:
是我用英文没讲明白嘛。我的意思是，使用 iframe 将视频嵌入到网站中并不需要作者的授权。因为我们“没有使用”作者的视频资源（这是一种对影视作品的引用）。我在上面提供的链接里面已经对“分享”影视作品是否构成侵权作了相对明确的说明。

hanyujie2002 commented:
> 是我用英文没讲明白嘛。我的意思是，使用 iframe 将视频嵌入到网站中并不需要作者的授权。因为我们“没有使用”作者的视频资源（这是一种对影视作品的引用）。我在上面提供的链接里面已经对“分享”影视作品是否构成侵权作了相对明确的说明。

I am sorry but the situation of **embedding (not just referring to)** a video is not contained in that article

But according to [this article](https://techlawforum.nalsar.ac.in/is-embedding-a-youtube-video-legal/), it is always legal to embed a Youtube video, as long as the video itself doesn't violate any copyright.

hanyujie2002 commented:
Thank you for your patient, I will modify the link back

hanyujie2002 committed:
Apply suggestions from code review

yin1999 committed:
Merge branch 'main' into other_embed_tech

yin1999 committed:
replace the Bing map

The Bing map requires to login for getting iframe code. Replace this with the free, and open geographic map - OpenStreetMap

yin1999 committed:
format

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -197,73 +195,74 @@ textarea.onkeyup = function () {
+- [`border: none`](/zh-CN/docs/Web/CSS/border)
+  - ：如果使用此属性，`<iframe>` 将不会显示周围的边框。否则，默认情况下，浏览器会显示 `<iframe>` 的边框（通常我们不希望显示边框）。
+- [`allowfullscreen`](/zh-CN/docs/Web/HTML/Element/iframe#allowfullscreen)
+  - : 如果设置，`<iframe>`则可以通过[全屏 API](/zh-CN/docs/Web/API/Fullscreen_API) 放置在全屏模式中（稍微超出本文的范围）。
```
```suggestion
  - : 如果设置，则可以通过[全屏 API](/zh-CN/docs/Web/API/Fullscreen_API) 将 `<iframe>` 置于全屏模式（稍微超出本文的范围）。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -197,73 +195,74 @@ textarea.onkeyup = function () {
 
-以上我们提到了安全问题——现在我们来详细介绍一下这一点。我们并不期望你第一次就能完全理解所有内容;
-我们只想让你意识到这一问题，在你更有经验并开始考虑在你的实验和工作中使用 `<iframe>` 时为你提供参考。此外，没有必要害怕和不使用 `<iframe>`——你只需要谨慎一点。继续看下去吧...
+以上我们提到了安全问题——现在我们来详细介绍一下这一点。你第一次并不需要完全理解所有内容；我们只想让你意识到这一问题，然后在你更有经验并开始考虑在你的实验和工作中使用 `<iframe>` 时为你提供参考。此外，没有必要害怕和拒绝使用 `<iframe>`——你只需要谨慎一点。继续看下去吧...
```
```suggestion
我们在前面提到了安全问题——现在我们来详细介绍一下这一点。你第一次并不需要完全理解所有内容；我们只想让你意识到这一问题，然后在你更有经验并开始考虑在你的实验和工作中使用 `<iframe>` 时为你提供参考。此外，没有必要害怕和拒绝使用 `<iframe>`——你只需要谨慎一点。继续看下去吧……
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -197,73 +195,74 @@ textarea.onkeyup = function () {
 
-浏览器制造商和 Web 开发人员了解到网络上的坏人（通常被称为**黑客**，或更准确地说是**破解者**），如果他们试图恶意修改你的网页或欺骗人们进行不想做的事情时常把 iframe
-作为共同的攻击目标（官方术语：**攻击向量**），例如显示用户名和密码等敏感信息。因此，规范工程师和浏览器开发人员已经开发了各种安全机制，使`<iframe>`更加安全，这有些最佳方案值得我们考虑——我们将在下面介绍其中的一些。
+浏览器制造商和 Web 开发人员已经深刻认识到，如果 Web 上的恶意人士（通常成为**黑客**，或者更准确地说，**骇客**）试图发起攻击，或者欺骗人们做一些他们不想做的事情，比如泄露敏感信息（如用户名和密码），那么 iframe 是常见目标（正式术语：**攻击向量**）。因此，规范工程师和浏览器开发人员已经开发了各种安全机制，以使 `<iframe>` 更安全，并且还有一些最佳实践需要考虑——我们将在下面介绍其中一些。
```
骇客是台湾地区对 hacker 的译法，国内就翻译为黑客。

```suggestion
浏览器制造商和 Web 开发人员已经深刻认识到，如果 Web 上的恶意人士（通常成为**黑客**，或者更准确地说，**破解者**）试图发起攻击，或者欺骗人们做一些他们不想做的事情，比如泄露敏感信息（如用户名和密码），那么 iframe 是常见目标（正式术语：**攻击向量**）。因此，规范工程师和浏览器开发人员已经开发了各种安全机制，以使 `<iframe>` 更安全，并且还有一些最佳实践需要考虑——我们将在下面介绍其中一些。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -197,73 +195,74 @@ textarea.onkeyup = function () {
 
-> **备注：** [单击劫持](https://en.wikipedia.org/wiki/Clickjacking)是一种常见的 iframe 攻击，黑客将隐藏的 iframe
-> 嵌入到你的文档中（或将你的文档嵌入到他们自己的恶意网站），并使用它来捕获用户的交互。这是误导用户或窃取敏感数据的常见方式。
+> **备注：** [单击劫持](/zh-CN/docs/Glossary/Clickjacking)是一种常见的 iframe 攻击，黑客将一个不可见的 iframe 嵌入到你的文档中（或将你的文档嵌入到他们自己的恶意网站），并使用它来获取用户的交互。这是一种常见的误导用户或窃取敏感数据的方式。
```
https://zh.wikipedia.org/wiki/%E7%82%B9%E5%87%BB%E5%8A%AB%E6%8C%81

```suggestion
> **备注：** [点击劫持](/zh-CN/docs/Glossary/Clickjacking)是一种常见的 iframe 攻击，黑客将一个不可见的 iframe 嵌入到你的文档中（或将你的文档嵌入到他们自己的恶意网站），并使用它来获取用户的交互。这是一种常见的误导用户或窃取敏感数据的方式。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -197,73 +195,74 @@ textarea.onkeyup = function () {
-的开发人员已经在网站页面的服务器上设置了一个不允许被嵌入到`<iframe>`的设置（请参阅[配置 CSP 指令](#配置_csp_指令)）这是有必要的——整个
-MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其嵌入到你的网站上并将其声称为自己的内容，或尝试通过单击劫持来窃取数据，这都是非常糟糕的事情。此外，如果每个人都这样做，所有额外的带宽将花费 Mozilla
-很多资金。
+一个简单的例子——尝试在浏览器中加载上面的例子——你也可以 [在 Github 上找到它](https://mdn.github.io/learning-area/html/multimedia-and-embedding/other-embedding-technologies/iframe-detail.html)（[参见源代码](https://github.com/mdn/learning-area/blob/gh-pages/html/multimedia-and-embedding/other-embedding-technologies/iframe-detail.html)）。与你期望的页面不同，你可能会看到某种类型的消息，内容类似于“无法打开此页面”，如果你查看[浏览器开发工具](/zh-CN/docs/Learn/Common_questions/Tools_and_ ...
```
```suggestion
一个简单的例子——尝试在浏览器中加载上面的例子——你也可以 [在 Github 上找到它](https://mdn.github.io/learning-area/html/multimedia-and-embedding/other-embedding-technologies/iframe-detail.html)（[参见源代码](https://github.com/mdn/learning-area/blob/gh-pages/html/multimedia-and-embedding/other-embedding-technologies/iframe-detail.html)）。与你期望的页面不同，你可能会看到某种类型的消息，内容类似于“无法打开此页面”，如果你查看[浏览器开发工具](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/What_are_browser_developer_tools)中的*控制台*，你将看到一条信息消息告知你原因。在 Firefox 中，消息类似于*由于“X-Frame-Options”指令设置为“DENY”，因此无法在 iframe 中加载“https\://developer.mozilla.org/zh-CN/docs/Glossary”*。这是因为构建 MDN 的开发人员在服务器上设置了一个选项，禁止将网站页面嵌入到 `<iframe>` 中（参见下面的[配置 CSP 指令](#配置_scp_指令)）。这是有原因的——整个 MDN 页面在其他页面中嵌入并不合适，除非你希望将它们嵌入到你的网站上并将它们声明为你自己的页面——或者通过[点击劫持](/zh-CN/docs/Glossary/Clickjacking)来尝试窃取数据，这两者都是非常糟糕的行为。而且，如果每个人这样做，额外的带宽成本将给 Mozilla 带来很大压力。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
 
-> **备注：** [Github 页面](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/Using_GitHub_pages)允许默认情况下通过 HTTPS
-> 提供内容，因此对托管内容很有用。如果你正在使用不同的托管，并且不确定，请向你的托管服务商询问。
+> **备注：** [Github pages](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/Using_GitHub_pages)默认情况下通过 HTTPS 提供内容。
+> 如果你正在使用其他的托管，请向你的托管服务商询问他们提供为 HTTPS 提供的支持。
```
这是个可选项（对于自定义域名）：

![image](https://github.com/mdn/translated-content/assets/15844309/cb76bce0-9f1d-41b6-8735-7470d9355f4e)

```suggestion
> **备注：** [Github pages](/zh-CN/docs/Learn/Common_questions/Tools_and_setup/Using_GitHub_pages) 允许默认情况下通过 HTTPS 提供内容。如果你正在使用其他的托管，请向你的托管服务商确定他们为基于 HTTPS 的内容托管提供哪些支持。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
 #### 始终使用 `sandbox` 属性
 
-想尽可能减少攻击者在你的网站上做坏事的机会，那么你应该给嵌入的内容仅能完成自己工作的权限。当然，这也适用于你自己的内容。一个允许包含在其里的代码以适当的方式执行或者用于测试，但不能对其他代码库（意外或恶意）造成任何损害的容器称为[沙盒](<https://en.wikipedia.org/wiki/Sandbox_(computer_security)>)。
+想尽可能减少攻击者在你的网站上做坏事的机会，那么你应该仅给嵌入的内容*完成自己工作所需的权限*。当然，这也适用于你自己的内容。[沙盒](<https://en.wikipedia.org/wiki/Sandbox_(computer_security)>)技术为此而诞生，沙盒是一个容器，其仅允许包含在内部的代码以适当的方式执行（或者用于测试），而无法对其他代码库（意外或恶意）造成任何损害。
```
```suggestion
想尽可能减少攻击者在你的网站上做坏事的机会，那么你应该仅给嵌入的内容*完成自己工作所需的权限*。当然，这也适用于你自己的内容。[沙盒](<https://zh.wikipedia.org/wiki/沙盒_(電腦安全)>)技术为此而诞生，沙盒是一个容器，其仅允许包含在内部的代码以适当的方式执行（或者用于测试），而无法对其他代码库（意外或恶意）造成任何损害。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
+想尽可能减少攻击者在你的网站上做坏事的机会，那么你应该仅给嵌入的内容*完成自己工作所需的权限*。当然，这也适用于你自己的内容。[沙盒](<https://en.wikipedia.org/wiki/Sandbox_(computer_security)>)技术为此而诞生，沙盒是一个容器，其仅允许包含在内部的代码以适当的方式执行（或者用于测试），而无法对其他代码库（意外或恶意）造成任何损害。
 
-未沙盒化（Unsandboxed）内容可以做得太多（执行 JavaScript，提交表单，弹出窗口等）默认情况下，你应该使用没有参数的 `sandbox` 属性来强制执行所有可用的限制，如我们前面的示例所示。
+未沙盒化（Unsandboxed）的内容可以执行 JavaScript，提交表单，弹出窗口等。默认情况下，你应该使用没有参数的 `sandbox` 属性来强制执行所有可用的限制，如我们前面的示例所示。
```
```suggestion
未沙盒化的内容可以执行 JavaScript、提交表单、弹出窗口等。默认情况下，你应该使用没有参数的 `sandbox` 属性来强制执行所有可用的限制，如我们前面的示例所示。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
+未沙盒化（Unsandboxed）的内容可以执行 JavaScript，提交表单，弹出窗口等。默认情况下，你应该使用没有参数的 `sandbox` 属性来强制执行所有可用的限制，如我们前面的示例所示。
 
-如果绝对需要，你可以逐个添加权限（`sandbox=""`属性值内）——请参阅 [`sandbox`](/zh-CN/docs/Web/HTML/Element/iframe#attr-sandbox) 所有可用选项的参考条目。其中重要的一点是，你*永远不*应该同时添加 `allow-scripts` 和 `allow-same-origin` 到你的 `sandbox` 属性中——在这种情况下，嵌入式内容可以绕过阻止站点执行脚本的同源安全策略，并使用 JavaScript 完全关闭沙盒。
+如果绝对需要某些权限，你可以逐个添加权限（`sandbox=""` 属性值内）——请参阅 [`sandbox`](/zh-CN/docs/Web/HTML/Element/iframe#sandbox) 所有可用选项的参考条目。其中重要的一点是，你*永远不*应该同时添加 `allow-scripts` 和 `allow-same-origin` 到你的 `sandbox` 属性中——在这种情况下，嵌入的内容可以绕过阻止站点执行脚本的[同源安全策略](/zh-CN/docs/Glossary/Same-origin_policy)，并使用 JavaScript 完全关闭沙盒。
```
```suggestion
如果绝对需要某些权限，你可以（在 `sandbox=""` 属性值内）逐个添加它们——请参阅 [`sandbox`](/zh-CN/docs/Web/HTML/Element/iframe#sandbox) 所有可用选项的参考条目。其中重要的一点是，你*永远不*应该同时添加 `allow-scripts` 和 `allow-same-origin` 到你的 `sandbox` 属性中——在这种情况下，嵌入的内容可以绕过阻止站点执行脚本的[同源安全策略](/zh-CN/docs/Glossary/Same-origin_policy)，并使用 JavaScript 完全关闭沙盒。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
-{{Glossary("CSP")}} 代表 **[内容安全策略](/zh-CN/docs/Web/HTTP/CSP)**，它提供[一组 HTTP 标头](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy)（由 web 服务器发送时与元数据一起发送的元数据），旨在提高 HTML
-文档的安全性。在 `<iframe>` 的安全性方面，你可以[_将服务器配置为发送适当的 `X-Frame-Options`
-标题_](/zh-CN/docs/Web/HTTP/Headers/X-Frame-Options)。这样做可以防止其他网站在其网页中嵌入你的内容（这将导致[点击劫持](https://zh.wikipedia.org/wiki/点击劫持)和一系列其他攻击），正如我们之前看到的那样，MDN 开发人员已经做了这些工作。
+{{Glossary("CSP")}} 全称是**[内容安全策略](/zh-CN/docs/Web/HTTP/CSP)**，它提供了[一组 HTTP 标头](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy)（访问 web 服务器时与网页一起发送的元数据），旨在提高 HTML 文档的安全性。在 `<iframe>` 的安全性方面，你可以*[配置服务器发送适当的 `X-Frame-Options` 标头](/zh-CN/docs/Web/HTTP/Headers/X-Frame-Options)*。这样可以防止其他网站在其网页中嵌入你的内容（这将导致[点击劫持](https://zh.wikipedia.org/wiki/点击劫持)和一系列其他攻击），正如我们之前看到的那样，MDN 开发人员已经做了这些工作。
```
![image](https://github.com/mdn/translated-content/assets/15844309/8a12515e-b0a5-42f1-8de3-f998f17eb5b2)

```suggestion
{{Glossary("CSP")}} 全称是[**内容安全策略**](/zh-CN/docs/Web/HTTP/CSP)，它提供了[一组 HTTP 标头](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy)（访问 web 服务器时与网页一起发送的元数据），旨在提高 HTML 文档的安全性。在 `<iframe>` 的安全性方面，你可以[_配置服务器发送适当的 `X-Frame-Options` 标头_](/zh-CN/docs/Web/HTTP/Headers/X-Frame-Options)。这样可以防止其他网站在其网页中嵌入你的内容（这将导致[点击劫持](https://zh.wikipedia.org/wiki/点击劫持)和一系列其他攻击），正如我们之前看到的那样，MDN 的开发人员已经做了这些工作。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
+{{Glossary("CSP")}} 全称是**[内容安全策略](/zh-CN/docs/Web/HTTP/CSP)**，它提供了[一组 HTTP 标头](/zh-CN/docs/Web/HTTP/Headers/Content-Security-Policy)（访问 web 服务器时与网页一起发送的元数据），旨在提高 HTML 文档的安全性。在 `<iframe>` 的安全性方面，你可以*[配置服务器发送适当的 `X-Frame-Options` 标头](/zh-CN/docs/Web/HTTP/Headers/X-Frame-Options)*。这样可以防止其他网站在其网页中嵌入你的内容（这将导致[点击劫持](https://zh.wikipedia.org/wiki/点击劫持)和一系列其他攻击），正如我们之前看到的那样，MDN 开发人员已经做了这些工作。
 
-> **备注：** 你可以阅读 Frederik Braun 的帖子[在 X-Frame-Options 安全性标头上](https://blog.mozilla.org/security/2013/12/12/on-the-x-frame-options-security-header/)来获取有关此主题的更多背景信息。显然，在这篇文章中已经解释得很清楚了。
+> **备注：** 你可以阅读 Frederik Braun 的帖子[论 X-Frame-Options 安全性标头](https://blog.mozilla.org/security/2013/12/12/on-the-x-frame-options-security-header/)来获取有关此主题的更多背景信息。其已经超出了本文章内容范围，故无法展开。
```
```suggestion
> **备注：** 你可以阅读 Frederik Braun 的帖子[论 X-Frame-Options 安全性标头](https://blog.mozilla.org/security/2013/12/12/on-the-x-frame-options-security-header/)来获取有关此主题的更多背景信息。显然，其已经超出了本文所解释内容的范围。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
-| 由插件控制的框的高度和宽度（以 CSS 像素为单位）            | [`height`](/zh-CN/docs/Web/HTML/Element/embed#height) [`width`](/zh-CN/docs/Web/HTML/Element/embed#width) | [`height`](/zh-CN/docs/Web/HTML/Element/object#height) [`width`](/zh-CN/docs/Web/HTML/Element/object#width) |
-| 名称和值，将插件作为参数提供                               | 具有这些名称和值的 ad hoc 属性                                                                            | 单标签[`<param>`](/zh-CN/docs/Web/HTML/Element/param)元素，包含在内`<object>`                               |
-| 独立的 HTML 内容作为不可用资源的回退                       | 不支持（`<noembed>`已过时）                                                                               | 包含在元素`<object>`之后`<param>`                                                                           |
+在过去，这些元素也用于嵌入由浏览器{{Glossary("Plugin", "插件")}}处理的内容如 {{Glossary("Adobe Flash")}}，但这个技术现在已经废弃了，不为现代浏览器所支持。
```
```suggestion
在过去，这些元素也用于嵌入由浏览器{{Glossary("Plugin", "插件")}}（如 {{Glossary("Adobe Flash")}}，但这个技术现在已经废弃了，不为现代浏览器所支持）处理的内容。
```

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -272,113 +271,107 @@ MDN 页面被嵌入在其他页面中没有多大意义，除非你想要将其
+    </tr>
+    <tr>
+      <td>用作后备资源的独立的 HTML 内容，以防资源不可用</td>
+      <td>不受支持（<code>&#x3C;noembed></code> 已废弃）</td>
```
```suggestion
      <td>不受支持（<code>&#x3C;noembed></code> 已过时）</td>
```

yin1999 reviewed (commented):
一些建议

hanyujie2002 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -19,84 +20,69 @@ slug: Learn/HTML/Multimedia_and_embedding/Other_embedding_technologies
 > **备注：** 考虑到中国大陆学习者可能无法访问英文原文的某些示例网站，下文的部分示例已经替换为更易于访问和实践的网站。
 
-在这篇文章中，我们将直接进入自主学习部分，让你立即体会到嵌入技术的实用性。大家都非常熟悉 [Bilibili](https://www.bilibili.com/)，但很多人不了解它所提供的一些分享功能。让我们来看看 Bilibili 如何让我们通过 [`<iframe>`](/zh-CN/docs/Web/HTML/Element/iframe) 在页面中嵌入喜欢的视频。
+在这篇文章中，我们将直接进入动手练习部分，让你立即体会到嵌入技术的实用性。大家都非常熟悉 [Bilibili](https://www.bilibili.com/)，但很多人不了解它所提供的一些分享功能。让我们来看看如何借助 Bilibili 提供的 {{htmlelement("iframe")}} 在页面中嵌入喜欢的视频。
```
```suggestion
在这篇文章中，我们将直接进入动手练习部分，让你立即体会到嵌入技术的实用性。大家都非常熟悉 [Bilibili](https://www.bilibili.com/)，但很多人不了解它所提供的一些分享功能。让我们来看看如何借助 {{htmlelement("iframe")}}，将 Bilibili 上的视频嵌入到页面中。
```

hanyujie2002 committed:
Apply suggestions from code review

Co-authored-by: A1lo <yin199909@aliyun.com>

hanyujie2002 committed:
update translation

yin1999 commented on files/zh-cn/learn/html/multimedia_and_embedding/other_embedding_technologies/index.md:
```diff
@@ -19,84 +20,69 @@ slug: Learn/HTML/Multimedia_and_embedding/Other_embedding_technologies
-一小段时间之后（20 世纪 90 年代末，21 世纪初），插件技术变得非常受欢迎，例如 [Java Applet](/zh-CN/docs/Glossary/Java) 和
-[Flash](/zh-CN/docs/Glossary/Adobe_Flash)——这些技术允许网络开发者将丰富的内容嵌入到网页中，例如视频和动画等，这些内容不能通过 HTML
-单独实现。嵌入这些技术是通过诸如 [`<object>`](/zh-CN/docs/Web/HTML/Element/object) 和较少使用的 [`<embed>`](/zh-CN/docs/Web/HTML/Element/embed) 元素来实现的，当时它们非常有用。由于许多问题，包括无障碍、安全性、文件大小等，它们已经过时了; 如今，大多数移动设备不再支持这些插件，桌面端也逐渐不再支持。
+一小段时间之后（20 世纪 90 年代末，21 世纪初），插件技术开始流行起来，例如 [Java Applet](/zh-CN/docs/Glossary/Java) 和 [Flash](/zh-CN/docs/Glossary/Adobe_Flash)——web 开发者借助这些技术将丰富的内容（例如视频和动画等）嵌入到网页中，这些内容不能通过 HTML 单独实现。嵌入这些技术是通过诸如 {{htmlelement("object")}} 和较少使用的 {{htmlelement("embed")}} 元素来实现的，当时它们非常有用。由于包括无障碍、安全性、文件大小在内的许多问题，它们现在已经被淘汰了。如今，大多数浏览器已不再支持 Flash 等插件。
```
```suggestion
不久之后（20 世纪 90 年代末，21 世纪初），插件技术开始流行起来，例如 [Java Applet](/zh-CN/docs/Glossary/Java) 和 [Flash](/zh-CN/docs/Glossary/Adobe_Flash)——web 开发者借助这些技术将丰富的内容（例如视频和动画等）嵌入到网页中，这些内容不能通过 HTML 单独实现。嵌入这些技术是通过诸如 {{htmlelement("object")}} 和较少使用的 {{htmlelement("embed")}} 元素来实现的，当时它们非常有用。由于包括无障碍、安全性、文件大小在内的许多问题，它们现在已经被淘汰了。如今，大多数浏览器已不再支持 Flash 等插件。
```

yin1999 reviewed (commented):
1. 我是特意将加粗仅应用到“框架”一词之上的，你可以看一下预览页面（因此我也很推荐大家可以对照着预览页面再过一遍内容，这样很容易发现这个问题）
    ![image](https://github.com/mdn/translated-content/assets/15844309/b7a010c8-6e33-4fb7-b2ab-a299250ba53e)
2. 个别词汇的翻译还需要注意一下，例如，我们不翻译“Web”

yin1999 committed:
Apply suggestions from code review

yin1999 committed:
Update index.md

yin1999 reviewed (approved):
感谢 @hanyujie2002
